### DISCUSSION TITLE
数据集支持

### DISCUSSION THREAD

**Discussion by w1125** [open-mmlab/mmocr#1615] [Q&A]
文本检测模型如何支持MSRA-TD500数据集训练？

**Comment by gaotongxiao** [comment]
We have not supported this dataset yet. But essentially you only have to convert the dataset annotation to [`MMOCR format`](https://mmocr.readthedocs.io/zh_CN/dev-1.x/migration/dataset.html#id6), and refer to the annotation file in [config file](https://mmocr.readthedocs.io/zh_CN/dev-1.x/user_guides/config.html#id10), then everything should work like a charm.
